packages/react/src/progress/progress.styles.ts

```diff
@@ -14,7 +14,7 @@ export const StyledProgress = styled("div", {
   padding: 0,
   width: "100%",
   position: "relative",
-  overflow: "visible",
+  overflow: "auto",
```

Comment:
Hey @wangly19 thanks for your contribution. 

In my opinion, `overflow: "hidden"` would be more suitable.

Reply:
I've revised it.